iothub/service/src/ExportImportDevice.cs

```diff
@@ -128,17 +129,29 @@ public string TwinETag
         public TwinCollection Tags { get; set; }
 
         /// <summary>
-        /// Desired and Reported property bags
+        /// Desired and reported property bags
         /// </summary>
         [JsonProperty(PropertyName = "properties", NullValueHandling = NullValueHandling.Ignore)]
         public PropertyContainer Properties { get; set; }
 
         /// <summary>
-        /// Status of Capabilities enabled on the device
+        /// Status of capabilities enabled on the device
         /// </summary>
         [JsonProperty(PropertyName = "capabilities", NullValueHandling = NullValueHandling.Ignore)]
         public DeviceCapabilities Capabilities { get; set; }
 
+        /// <summary>
+        /// The scope of the device. For edge devices, this is auto-generated and immutable. For leaf devices, set this to create child/parent
+        /// relationship.
+        /// </summary>
+        /// <remarks>
+        /// For leaf devices, the value to set a parent edge device can be retrieved from the parent edge device's device scope property.
+        ///
+        /// For more information, see <see href="https://docs.microsoft.com/azure/iot-edge/iot-edge-as-gateway?view=iotedge-2020-11#parent-and-child-relationships"/>.
+        /// </remarks>
+        [JsonProperty(PropertyName = "deviceScope", NullValueHandling = NullValueHandling.Include)]
+        public string DeviceScope { get; set; }
```

Comment:
The name in the `Device` object was `Scope` even though the on-the-wire-property is `deviceScope`.

When I added device scope to Twin when working on parent scopes, I named it `DeviceScope`, because in context of `Twin` it required the full name. I feel the same is true if `ExportImportDevice`.

I wish we hadn't named it `Device.Scope`, but it is too late to reconsider that.